src/uct/sm/mm/mm_posix.c

```diff
@@ -16,8 +16,18 @@
 #define UCT_MM_POSIX_MMAP_PROT      (PROT_READ | PROT_WRITE)
 #define UCT_MM_POSIX_HUGETLB        UCS_BIT(0)
 
+static ucs_config_field_t uct_posix_pd_config_table[] = {
+  {"MM_", "", NULL,
+   ucs_offsetof(uct_posix_pd_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_mm_pd_config_table)},
+
+  {"DIR", "/dev/shm", "The path to the backing file",
```

Comment:
i didnt use it yet, just set this value. 
same is true for hugetlb mode in mm_config. so that it wont be empty...